src/Tribe/Service_Providers/Onboarding.php

```diff
@@ -0,0 +1,140 @@
+<?php
+namespace Tribe\Service_Providers;
+
+/**
+ * Class Onboarding
+ *
+ * @since TBD
+ *
+ * Handles the registration and creation of our async process handlers.
+ */
+class Onboarding extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * The Onboarding assets group identifier.
+	 *
+	 * @var string
+	 */
+	private static $assets_group_id = 'tribe-onboarding';
```

Comment:
let's make this public, we do that in other places. also, let's call it `$group_key`, mimicking the same in TEC.